navigation/ca_cartographer/launch/pure_localization.launch

```diff
@@ -0,0 +1,22 @@
+<launch>
+  <param name="/use_sim_time" value="true" />
+
+  <arg name="load_state_filename" default="$(find ca_cartographer)/house_map/house_map.pbstream" />
+
+  <node name="cartographer_node" pkg="cartographer_ros"
+      type="cartographer_node" args="
+          -configuration_directory $(find ca_cartographer)/config
+          -configuration_basename ca_localization_2d.lua
+          -load_state_filename $(arg load_state_filename)"
+      output="screen">
+    <remap from="scan" to="create1/laser/scan" />
```

Comment:
Remove "create1" from remappings and add this:

```xml
<arg name="robot_id" default="$(optenv ID 1)"/>
<arg name="ns"       default="create$(arg robot_id)"/>

<node ns="$(arg ns)" ....>
</node>
```
```

Reply:
Although Cartographer has issues loading in a namespace, I used the ns parameter to concatenate the remaps and make it generic.